server/src/@common/dto/map/map-info.dto.ts

```diff
@@ -45,3 +45,20 @@ export class CreateMapInfoDto extends PickType(MapInfoDto, [
     'numTracks',
     'creationDate'
 ] as const) {}
+
+export class UpdateMapInfoDto {
+    @ApiProperty({ description: 'The new description to set', type: String })
+    @IsOptional()
+    @IsString()
+    description: string;
+
+    @ApiProperty({ description: 'The new youtube ID to set', type: String })
+    @IsOptional()
+    @Matches(/^[\w_-]{11}$/)
+    youtubeID: string;
+
+    @ApiProperty( { description: 'The new creation date to set', type: Date })
+    @IsOptional()
+    @IsDateString()
+    creationDate: Date;
+}
```

Comment:
I think it's best we extend the existing types on the MapInfoDto - I don't even mind that it won't have a description specifically about updating. So we can use [NestJS's type mapping](https://docs.nestjs.com/openapi/mapped-types) and simply do something like
```ts
export class UpdateMapInfoDto extends PartialType(PickType(MapInfoDto, ['description', 'youtubeID', 'creationDate'] as const)) {}
```